zebrad/src/components/rpc/config.rs

```diff
@@ -0,0 +1,27 @@
+//! User-configurable RPC parameters.
+
+use std::net::SocketAddr;
+
+use serde::{Deserialize, Serialize};
+
+/// RPC configuration section.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(deny_unknown_fields, default)]
+pub struct Config {
+    /// IP address and port for the RPC server.
+    ///
+    /// Note: RPC server will not start if this is `None`.
+    pub listen_addr: Option<SocketAddr>,
+}
+
+impl Default for Config {
+    fn default() -> Self {
+        Self {
+            listen_addr: Some(
+                "127.0.0.1:8232"
+                    .parse()
+                    .expect("Hardcoded address should be parseable"),
```

Comment:
Security: in the design meeting, we decided to have the RPC listener off by default, so that users know which ports they need to protect.

But it would be good to mention the default `127.0.0.1:8232` in the docs, so that people use a secure setting.

Reply:
Turning this port off by default should also fix port conflicts in the tests.

Reply:
Ok, i changed this at https://github.com/ZcashFoundation/zebra/pull/3589/commits/6ec258a55e165685ce887428f144e07a2e4888ec however, by some reason, this does not fix the acceptance tests port conflict issues.